### PR TITLE
LPS-132017 Add Unit Tests

### PR DESCRIPTION
**Pull request by mateussgn**
Related issue: https://issues.liferay.com/browse/LPS-132017

### CODE REVIEW

**Comment by guilhermedcamacho** [comment]
Just started reviewing :)

**Comment by choliveira84** [comment]
Just started reviewing :)

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/test/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessorTest.java [hunk 1]
```diff
@@ -85,6 +85,32 @@ public void testGetDDMFormCustomField() {
 		Assert.assertEquals("custom_field", ddmFormField.getType());
 	}
 
+	@Test
+	public void testGetDDMFormFieldRequiredErrorMessage() {
```
Comment (guilhermedcamacho) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/test/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessorTest.java:89]: Hey @mateussgn ,

What do you think to just check the new property in the existing method testGetDDMFormTextField? It already checks other similar properties like confirmationErrorMessage, confirmationLabel and so on.

**Review by guilhermedcamacho — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/test/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessorTest.java (hunk 1 above)
Comment (mateussgn) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/test/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessorTest.java]: Hey @guilhermedcamacho, I've already made the requested changes and pushed them up, thanks for advising me! 😃

**Comment by guilhermedcamacho** [comment]
Just started reviewing :)

**Comment by guilhermedcamacho** [comment]
Just started reviewing :)

**Comment by guilhermedcamacho** [comment]
Resent in #502

**Comment by guilhermedcamacho** [comment]
Just started reviewing :)

**Comment by guilhermedcamacho** [comment]
Just started reviewing :)

**Pull request closed without merging**